src/playback/qml/MuseScore/Playback/internal/AudioResourceControl.qml

```diff
@@ -241,17 +235,17 @@ Item {
         Loader {
             id: selectorLoader
 
-            Layout.preferredWidth: selectorLoader.active ? 24 : 0
+            Layout.fillWidth: !titleLoader.visible
+            Layout.preferredWidth: root.height
             Layout.preferredHeight: root.height
             Layout.alignment: Qt.AlignRight
 
-            active: (root.showAdditionalButtons || root.resourcePickingActive) && root.supportsMenu
+            visible: root.showAdditionalButtons && root.supportsMenu
```

Comment:
not quite sure about this change, seems redundant

Reply:
Do you mean the change from `active` to `visible`? Mainly because I think it is semantically more correct, but also because Layout automatically excludes items that are invisible when it calculates the position of other items. So we don't need to set `Layout.preferredWidth` to zero manually (using property binding). That has the advantage that we can bind other properties to `Layout.preferredWidth`. But you could also argue that it is no problem when `Layout.preferredWidth` is set to zero when the loader is inactive, because then the items that would bind properties to it are not rendered anyway... 

Or if you mean that I removed `|| root.resourcePickingActive`, that is because I moved that inside the `root.showAdditionalButtons` property, because I found that we're using `root.showAdditionalButtons || root.resourcePickingActive` again and again.

Reply:
Ha, haven't seen this "root.showAdditionalButtons" before, who made this change?

Looks fine to me anyway